map_view/display-device-location-with-autopan-modes/README.metadata.json

```diff
@@ -12,26 +12,24 @@
         "map",
```

Comment:
map is still showing here: is the script not working?

Reply:
Hey @Rachael-E , it looks like the README.metadata.json has been updated: https://github.com/Esri/arcgis-runtime-samples-java/blob/0570f41115e9efaa8231a7d71dfff529cb065ad7/map_view/display-device-location-with-autopan-modes/README.metadata.json